scale/utils.py

```diff
@@ -0,0 +1,155 @@
+#/usr/bin/python
+
+"""Kubernetes API access functions"""
+
+import requests
+import json
+import sys
+from settings import API_HOST, API_PORT
+import logging
+
+logging.getLogger("requests").setLevel(logging.WARNING)
+
+
+def generateUrl(host, port):
+    return "http://" + host + ':' + port + "/api/v1/"
+
+
+def getNodes():
```

Comment:
A lot of this is just recreating the kubernetes API client. We shouldn't be doing that...